oidc-controller/api/core/config.py

```diff
@@ -233,6 +233,9 @@ class GlobalConfig(BaseSettings):
     CONTROLLER_VARIABLE_SUBSTITUTION_OVERRIDE: str | None = os.environ.get(
         "CONTROLLER_VARIABLE_SUBSTITUTION_OVERRIDE"
     )
+    CONTROLLER_TEMPLATE_DIR: str = os.environ.get(
+        "CONTROLLER_TEMPLATE_DIR", "/etc/templates"
```

Comment:
Would it make sense to use `/etc/controller-config/templates` to keep all configs within the same directory? Any potential issues when mounting volumes for both the templates AND the index configurations? It might be possible to address this by creating the `/etc/controller-config` folder in the dockerfile with the right permission as an empty container and moving files into it during the build and/or via a mounted volume.

Reply:
Totally I'll make the change.